riseagain47/EPIC-Mail
# Ft add user to group #164725538

#### What does this PR do?
- This PR will create an endpoint to help adds a user(s) to a group

#### Description of Task to be completed?
Have the following:
- group model
- group controller 
- group route
- group unit and integration tests

#### How should this be manually tested?
1. User should start the server with `npm run dev-start`
2. Open Postman and makes a `POST` HTTP request to `localhost:5000/api/v2/groups/:id/users
3. User should receive a success message upon success and adds a user to a group or get a no group response.

#### Any background context you want to provide?
User should that the server is running before making the `POST` HTTP request

#### What are the relevant pivotal tracker stories?
- #164725538 - Feature - User can add user(s) to a group
